TotalImage.IO/FileSystems/FAT/FatFileSystem.cs

```diff
@@ -11,10 +11,13 @@ public abstract class FatFileSystem : FileSystem
     {
         protected readonly BiosParameterBlock _bpb;
         public BiosParameterBlock BiosParameterBlock => _bpb;
+        protected readonly bool _isSmall;
+        public bool IsSmall => _isSmall;
```

Comment:
I think we should move this to the FAT12 class and make all the properties affected by direntry size overridable.